source/develop/release-management/features/storage/wipe-volume-using-blkdiscard.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: WipeVolumeUsingBlkdiscard
+authors: ishaby
+wiki_title: Features/WipeVolumeUsingBlkdiscard
+feature_name: Wipe volume using blkdiscard
+feature_modules: vdsm
+feature_status: Work in progress for oVirt 4.1
+---
+
+# Wipe volume using blkdiscard
+
+## Summary
+
+### How volumes are wiped today
+When a disk on a block domain has its ***Wipe After Delete*** property enabled and it is removed, vdsm first wipes its volumes (writes zeros on them) and only then removes them.<br/>
+Today, vdsm uses the `dd` linux command to wipe volumes.
+
+### What can be improved
+The problem with using `dd` to wipe volumes is that it is **very slow** (~ 7 minutes to wipe a 10GB volume on netapp in a simple development environment).<br/>
+To zero volumes more efficiently, vdsm can use the `blkdiscard` command from the *util-linux* package, which can run up to ~ 10 times faster.
+
+### For whom is this feature useful
+This feature is useful for users that have their disks' *Wipe After Delete* property enabled and wish to wipe them faster when they are removed.
```

Comment:
Everyone always wants everything to run faster, that's tautology.
"This feature is useful for security-conscious users who mark WAD=true and use modern storage arrays that support XYZ"